client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -34,6 +34,7 @@ import IssuerEvidenceList from './evidence-list';
 import DisputeSummaryRow from './dispute-summary-row';
 import { DisputeSteps, InquirySteps } from './dispute-steps';
 import InlineNotice from 'components/inline-notice';
+import WCPaySettingsContext from '../../settings/wcpay-settings-context';
```

Comment:
Nit: use the `wcpay` import path rather than `../../` to improve readability.
```suggestion
import WCPaySettingsContext from 'wcpay/settings/wcpay-settings-context';
```